action/kafkaFeedWeb.js

```diff
@@ -61,6 +61,35 @@ function main(params) {
 
                     resolve(common.webResponse(statusCode, body));
                 });
+        } else if (params.__ow_method === "get") {
+            const triggerURL = common.getTriggerURL(params.authKey, params.endpoint, params.triggerName);
+
+            return common.verifyTriggerAuth(triggerURL)
+                .then(() => {
+                    db = new Database(params.DB_URL, params.DB_NAME);
+                    return db.getTrigger(params.triggerName);
+                })
+                .then((triggerDoc) => {
+                    var body = {
+                        config: {
+                            triggerName: triggerDoc.triggerName,
+                            topic: triggerDoc.topic,
+                            isJSONData: triggerDoc.isJSONData,
+                            isBinaryValue: triggerDoc.isBinaryValue,
+                            isBinaryKey: triggerDoc.isBinaryKey,
+                            isMessageHub: triggerDoc.isMessageHub,
+                            brokers: triggerDoc.brokers,
+                            kafka_admin_url: triggerDoc.kafka_admin_url,
+                            username: triggerDoc.username,
+                            password: triggerDoc.password
```

Comment:
Do we really want to return the password?

Reply:
at this point in time yes. the user needs to be authenticated. this is no different than fetching the service credentials for your instance of message hub from bluemix

Reply:
we will likely add flags in the future to adjust what is returned. @csantanapr wants to get this initial work out before focusing on that as this is highly desirable

Reply:
The code is not entirely duplicated. messageHubFeedWeb.js has a different method to validate parameters. Most of the shared code there is separated out into functions that is shared amongst the new files.

Reply:
@csantanapr, thoughts on exposing the password here?